airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/command/WriteConfiguration.kt

```diff
@@ -23,6 +26,9 @@ interface WriteConfiguration {
 
 @Singleton
 @Secondary
+// Groups of quasi-constants like these should be injected using @ConfigurationProperties,
+// it's so much more convenient!
+// See DefaultJdbcConstants for an example of what I mean.
```

Comment:
I tried using `@ConfigurationProperties` (specifically for the Destination Config) and found it less useful than just injecting the config object. Fewer magic strings everywhere.

Reply:
What do you mean by magic strings? see DefaultJdbcConstants which doesn't have any.